prow/config.yaml

```diff
@@ -640,6 +642,45 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  - name: pull-test-infra-bazel-scenario
+    context: pull-test-infra-bazel-scenario
+    branches:
+    - master
+    rerun_command: "/test pull-test-infra-bazel-scenario"
+    trigger: "((?m)^@k8s-bot pull-test-infra-bazel-scenario test this,?(\\s+|$)|(?m)^/test pull-test-infra-bazel-scenario,?(\\s+|$))"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170728-71064cf4
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--json"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /root/.cache
```

Comment:
also set `TEST_TMPDIR` to this

(we need this for all of the bazel scenario jobs, I think)